src/storage/txn/scheduler.rs

```diff
@@ -67,6 +67,12 @@ pub const GC_BATCH_SIZE: usize = 512;
 // The write batch will be around 32KB if we scan 256 keys each time.
 pub const RESOLVE_LOCK_BATCH_SIZE: usize = 256;
 
+/// After scanned how many versions for a key we should output a log for it.
```

Comment:
After the GC scan of a key, output a message to the log if there are at least this many versions of the key.